test/unit/test_exports/test_derived_quantities/test_point_value.py

```diff
@@ -0,0 +1,34 @@
+from festim import PointValue
+import fenics as f
+import pytest
+
+
+def test_title_H():
+    x = 1
+    field = "solute"
+    my_value = PointValue(field, x)
+    assert my_value.title == "{} value at [{}]".format(field, x)
+
+
+def test_title_T():
+    x = 1
+    field = "T"
+    my_value = PointValue(field, x)
+    assert my_value.title == "{} value at [{}]".format(field, x)
```

Comment:
We can merge these two tests using [pytest parametrized tests](https://docs.pytest.org/en/7.1.x/how-to/parametrize.html).

Reply:
Shall we modify all other tests (e.g. `test_total_volume`,  `test_total_surface`, etc.) in the same way?

Reply:
Ideally yes but let's do this in a separate dedicated PR.